app/pulp/app/tests/models/test_content.py

```diff
@@ -94,15 +94,6 @@ def test_reading(self):
                 self.assertEqual(artifact.file.read(), fp.read())
             artifact.file.close()
 
-    def test_publishing_repository(self):
-        self.associate()
-        self.add_artifacts()
-
-        # publishing
-        for content in (c.cast() for c in self.repository.content.filter(type=TYPE)):
-            for artifact in content.artifacts.all():
-                self.publish(artifact)
-
```

Comment:
This was throwing `django.db.utils.ProgrammingError: relation "pulp_boringcontent" does not exist` because writing ad-hoc models for testing isn't reliable because of niggling things exactly like this. BoringContent should go away, if we want testing models we need to actually write them and make sure they have migrations before tests run.

The test had no assertions or comments, so it wasn't obvious to me how to fix it.